### PR TITLE
feat: cdw-31-create-endpoint-to-add-users-to-database

### PR DESCRIPTION
**Pull request by ifasama**
as the title said and a verification to check whether username exists.

### CODE REVIEW

File: backend/src/api/v1/mentorShip/controllers/userController.ts [hunk 1]
```diff
@@ -30,3 +34,22 @@ export const getSingle = async (req: Request, res: Response) => {
...
+    })
+
+    /* {
+    "firstName": "John",
+    "lastName": "Rolfe",
+    "email": "mrjohn@mail.com",
+    "username": "johnro",
+    "password": "password",
+    "location": "Ontario"
+} */
```
Comment (jtabba) [inline backend/src/api/v1/mentorShip/controllers/userController.ts]: Remove commented code

File: backend/src/database/userDatabase.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { connectToDb } from "./db"
+/* import { connectToDb } from "./db"
```
Comment (jtabba) [inline backend/src/database/userDatabase.ts]: Unrequired comments

**Review by jtabba — CHANGES_REQUESTED** [review]
Minor changes requested - great job overall!

**Comment by ifasama** [comment]
I made the changes

**Review by jtabba — APPROVED** [review]

**Pull request merged**